.gitlab-ci.yml

```diff
@@ -46,8 +46,8 @@ Container Build:
 
     - podman build --format docker -t flux --build-arg BASE_IMAGE=base docker/flux
     - podman build --format docker -t parsl --build-arg BASE_IMAGE=flux docker/parsl
-    - podman build --format docker -t rp --build-arg BASE_IMAGE=parsl docker/rp
-    - podman build --format docker -t swift-t --build-arg BASE_IMAGE=rp docker/swift-t
+    - podman build --format docker -t rct --build-arg BASE_IMAGE=parsl docker/rct
+    - podman build --format docker -t swift-t --build-arg BASE_IMAGE=rct docker/swift-t
```

Comment:
I think we are aiming to remove the podman build part of the Gitlab CI, and focus on the spack tests. Are you able to add an rct test to `ci/tests/rct/test.sh`? I can rebase and adjust the rest of the Gitlab CI after this is merged

Reply:
I would keep added test for EnTK for GitHub Actions only, thus the currently available test `/ci/tests/rp/test.sh` can be continued to be used. Thus do you propose to update `rp` to `rct` in all directories under `/ci/tests`?

Reply:
It would be nice to run the additional tests on the Spack Build as well, but it makes sense to add that in a separate PR. I think this is good for this one, thanks!